src/main/java/appeng/client/render/FacadeDispatcherBakedModel.java

```diff
@@ -45,11 +45,13 @@
 public class FacadeDispatcherBakedModel extends DelegateBakedModel
 {
 	private final VertexFormat format;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "format" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)